django-backend/fecfiler/f3x_summaries/models.py

```diff
@@ -0,0 +1,135 @@
+    L17_other_federal_receipts_dividends_ytd = models.IntegerField(null=True, blank=True)
+    L18a_transfers_from_nonfederal_account_h3_ytd = models.IntegerField(null=True, blank=True)
+    L18b_transfers_from_nonfederal_levin_h5_ytd = models.IntegerField(null=True, blank=True)
+    L18c_total_nonfederal_transfers_18a_18b_ytd = models.IntegerField(null=True, blank=True)
+    L19_total_receipts_ytd = models.IntegerField(null=True, blank=True)
+    L20_total_federal_receipts_ytd = models.IntegerField(null=True, blank=True)
+    L21ai_federal_share_ytd = models.IntegerField(null=True, blank=True)
+    L21aii_nonfederal_share_ytd = models.IntegerField(null=True, blank=True)
+    L21b_other_federal_operating_expenditures_ytd = models.IntegerField(null=True, blank=True)
+    L21c_total_operating_expenditures_ytd = models.IntegerField(null=True, blank=True)
+    L22_transfers_to_affiliated_other_party_cmtes_ytd = models.IntegerField(null=True, blank=True)
+    L23_contributions_to_federal_candidates_cmtes_ytd = models.IntegerField(null=True, blank=True)
+    L24_independent_expenditures_ytd = models.IntegerField(null=True, blank=True)
+    L25_coordinated_expend_made_by_party_cmtes_ytd = models.IntegerField(null=True, blank=True)
+    L26_loan_repayments_made_ytd = models.IntegerField(null=True, blank=True)
+    L27_loans_made_ytd = models.IntegerField(null=True, blank=True)
+    L28a_individuals_persons_ytd = models.IntegerField(null=True, blank=True)
+    L28b_political_party_committees_ytd = models.IntegerField(null=True, blank=True)
+    L28c_other_political_committees_ytd = models.IntegerField(null=True, blank=True)
+    L28d_total_contributions_refunds_ytd = models.IntegerField(null=True, blank=True)
+    L29_other_disbursements_ytd = models.IntegerField(null=True, blank=True)
+    L30ai_shared_federal_activity_h6_fed_share_ytd = models.IntegerField(null=True, blank=True)
+    L30aii_shared_federal_activity_h6_nonfed_ytd = models.IntegerField(null=True, blank=True)
+    L30b_nonallocable_fed_election_activity_ytd = models.IntegerField(null=True, blank=True)
+    L30c_total_federal_election_activity_ytd = models.IntegerField(null=True, blank=True)
+    L31_total_disbursements_ytd = models.IntegerField(null=True, blank=True)
+    L32_total_federal_disbursements_ytd = models.IntegerField(null=True, blank=True)
+    L33_total_contributions_ytd = models.IntegerField(null=True, blank=True)
+    L34_total_contribution_refunds_ytd = models.IntegerField(null=True, blank=True)
+    L35_net_contributions_ytd = models.IntegerField(null=True, blank=True)
+    L36_total_federal_operating_expenditures_ytd = models.IntegerField(null=True, blank=True)
+    L37_offsets_to_operating_expenditures_ytd = models.IntegerField(null=True, blank=True)
+    L38_net_operating_expenditures_ytd = models.IntegerField(null=True, blank=True)
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
```

Comment:
I may have missed this in the acceptance criteria, but we need to do soft deletes. We'll need a "deleted" column of type boolean. Also, a "delete" method override that sets the "deleted" column to true on a delete call.